app/controllers/users_controller.rb

```diff
@@ -8,6 +8,11 @@ def show
   end
 
   def update
+    if avatar.present?
+      current_user.avatar.attach avatar
+      current_user.image_url = url_for(current_user.avatar)
```

Comment:
How does this deal with current images fetched from social providers? Can both types of images coexist?

Reply:
When a user changes its image, the one from the social provider is overwritten and get lost forever.

Maybe we can add a `default_image_url` with the social provider image, and a button in the UI to "reset" the avatar to the default one (in case there's any)

Reply:
I think we are just missing this and we are ready to go!